packages/x-charts/src/context/SurfacePropsProvider/SurfacePropsContext.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { Initializable } from '../context.types';
+import { SurfacePropsContextState } from './SurfaceProps.types';
+
+export const SurfacePropsContext = React.createContext<Initializable<SurfacePropsContextState>>({
+  isInitialized: false,
+  data: {
+    width: 0,
+    height: 0,
+  },
+});
```

Comment:
What about moving this to the DrawingProvider? This `width` is equal to the `left+width+right`. It could be renamed `svgWidth`

Reply:
We can probably just get it directly from the `DrawingProvider` in the `ChartsSurface` instead 🤔